docs/content/getting-started/faq.md

```diff
@@ -181,3 +181,19 @@ and the message should help in figuring out the mistake(s) in the configuration,
 
 When using the file provider,
 one easy way to check if the dynamic configuration is well-formed is to validate it with the [JSON Schema of the dynamic configuration](https://json.schemastore.org/traefik-v2-file-provider.json).
+
+## Why does Let's Encrypt wildcard certificate renewal/generation with DNS challenge fail?
+
+If you're trying to renew wildcard certificates, with DNS challenge,
+and you're getting errors such as:
+
+<pre class="highlight"><code class="language-text" style="white-space : pre-wrap !important;">time="2023-01-28T17:13:58+01:00" level=error msg="Error renewing certificate from LE: {subdomain.domain.name [*.subdomain.domain.name]}" providerName=letsencrypt.acme error="error: one or more domains had a problem:\n[*.subdomain.domain.name] [*.subdomain.domain.name] acme: error presenting token: gandiv5: unexpected authZone domain.name. for fqdn domain.name.\n[subdomain.domain.name] [subdomain.domain.name] acme: error presenting token: gandiv5: unexpected authZone domain.name. for fqdn domain.name.\n" ACME CA="https://acme-v02.api.letsencrypt.org/directory"</code></pre>
```

Comment:
```suggestion

```txt
level=error msg="Error renewing certificate from LE: {example.com [*.example.com]}" providerName=letsencrypt.acme error="error: one or more domains had a problem:\n[example.com] [example.com] acme: error presenting token: gandiv5: unexpected authZone domain.name. for fqdn example.com.\n""
```
```

Reply:
I understand the contents changes (and they're fair enough), but the style change does not achieve the same result.
With the custom style, we get a block of text that is automatically wrapped to the width of the available space, so it is actually readable for the users without having to scroll left and right.